### DISCUSSION TITLE
Dumb question about updates

### DISCUSSION THREAD

**Discussion by mateusmarasca** [LizardByte/community#219] [Sunshine] [answered]
Hello, guys! Could you help a newbie on how to update Sunshine when it says there is a new update?

**Comment by ReenigneArcher** [comment, marked as answer]
Just download the new version and install it.

**Reply by mateusmarasca** [reply]
Damn, that's too obvious for me to not have tought about. Will I keep my config and apps if I do this?
Thanks!

**Reply by ReenigneArcher** [reply]
Yes as long as you don't remove them during install... good idea to create a backup first.

**Comment by Eikwerno** [comment]
Is it possible to run the update while Sunshine is running? I am asking because my host running Sunshine is not connected to a screen but is only accessed via Moonshine. Updating the existing version would probably require stopping the running one which in that case interputs the connection with Moonshine. Or do I need to hook the host up with a screen temporarily to run the update?

**Reply by ReenigneArcher** [reply]
If you're running the Sunshine service, you can update while it's running. It will terminate the existing session if you're connected to one.
